src/core/utils/RbSettings.cpp

```diff
@@ -321,8 +321,8 @@ void RbSettings::writeUserSettings( void )
     path settings_file_name = user_dir / ".RevBayes.ini";
 
     std::ofstream writeStream( settings_file_name.string() );
-    assert( moduleDir == "modules" or is_directory(moduleDir) );
-    writeStream << "moduledir=" << moduleDir << std::endl;
+    assert( moduleDir.string() == "modules" or is_directory(moduleDir) );
+    writeStream << "moduledir=" << moduleDir.string() << std::endl;
```

Comment:
I think we want the quotes - we just don't want to add more quotes every time.  Quotes help to handle spaces or weird characters in the filename.

Instead of removing the quotes, I think we should just read the module dir from a stream, to match writing it to a stream.

Can you instead modify `RbSettings::setOption( )` to do the following:
```
        // Read from stream to handle quotes.
        std::istringstream input(value);
        input >> moduleDir;
```

I check that this works.

Reply:
OK, I tried your suggestion and that works too. I don't have a strong preference either way but can see that reading without quotes could be complicated. However, I think we separate the key-value pair by "=" and if we remove the trailing spaces, then a path with spaces but without quotes should still work (not tested).

Reply:
Yeah, I was thinking about a filename with trailing spaces.... probably pretty rare.